Marlin/SanityCheck.h

```diff
@@ -261,6 +261,15 @@
 #endif
 
 /**
+ * DHT Sensor option
+ */
+#if ENABLED(DHT_ENABLE)
+  #if DISABLED(ULTIPANEL)
+    #error "DHT Sensor option requires an LCD controller."
+  #endif
+#endif
```

Comment:
Should I move this to sanity check instead of automaticly enabling the `lcd_info_menu`?